### PR TITLE
Feat: Vibes app

### PR DESCRIPTION
**Pull request by josenriagu**
## Description

- update vibe app

## Checklist

- [x] I have read the **README** document
- [x] I have read the **CONTRIBUTING** document
- [ ] I have added tests to cover my changes
- [x] All new and existing tests passed
- [x] My commit message follows [Conventional Commit Guideline](https://conventionalcommits.org/)

### CODE REVIEW

File: ui/apps/profile/package.json [hunk 1]
```diff
@@ -45,14 +45,12 @@
...
     "react-i18next": "12.2.0",
     "@tanstack/react-query": "4.29.3",
-    "react-router-dom": "6.3.0",
     "single-spa-react": "6.0.1",
     "@tanstack/react-router": "1.16.6"
   },
   "devDependencies": {
     "@akashaorg/af-testing": "*",
-    "@akashaorg/typings": "*",
-    "@tanstack/router-devtools": "1.16.6"
```
Comment (quininez) [inline ui/apps/profile/package.json:55]: why remove tanstack devtools here?

File: ui/apps/vibes/src/utils/format-date.ts [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import dayjs from 'dayjs';
```
Comment (quininez) [inline ui/apps/vibes/src/utils/format-date.ts]: you can add this to the time.ts in design-system-core, so we have all the date formatting in one place

**Review by quininez — CHANGES_REQUESTED** [review]

File: ui/apps/profile/package.json (hunk 1 above)
Comment (josenriagu) [reply ui/apps/profile/package.json:55]: I have moved the devtools to topbar so it stays on for very route, so it is now redundant on the profile app

File: typings/src/ui/router.ts [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import { useApolloClient } from '@apollo/client';
+
+export type ApolloClient = ReturnType<typeof useApolloClient>;
```
Comment (didd) [inline typings/src/ui/router.ts]: No need to export this. We can use the class `ApolloClient` from the library as type `ApolloClient<object>`.
`ReturnType<typeof useApolloClient>` is the same as `ApolloClient<object>`

**Review by quininez — APPROVED** [review]

**Pull request merged**